bodhi/server/push.py

```diff
@@ -42,21 +42,15 @@
               help='Push updates with a specific request (default: testing,stable)')
 @click.option('--resume', help='Resume one or more previously failed pushes',
               is_flag=True, default=False)
-@click.option('--staging', help='Use the staging bodhi instance',
-              is_flag=True, default=False)
 @click.option('--username', prompt=True)
 @click.version_option(message='%(version)s')
 def push(username, cert_prefix, **kwargs):
     """Push builds out to the repositories."""
-    staging = kwargs.pop('staging')
     resume = kwargs.pop('resume')
 
     lockfiles = defaultdict(list)
     locked_updates = []
-    if staging:
-        locks = '/var/cache/bodhi/mashing/MASHING-*'
-    else:
-        locks = '/mnt/koji/mash/updates/MASHING-*'
+    locks = '%s/MASHING-*' % config.get('mash_dir')
```

Comment:
Yeah good call on killing ```--staging``` - since ```mash_dir``` is part of staging's config, it's not needed anyway.